### PR TITLE
Improve error handling in examples template code.

### PR DESCRIPTION
**Pull request by frankfliu**
*Issue #, if available:*

*Description of changes:*

**To run CI tests on your changes refer [README.md](ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/model_service_template/mxnet_vision_service.py [hunk 1]
```diff
@@ -25,6 +26,15 @@ class MXNetVisionService(MXNetModelService):
     """
 
     def preprocess(self, request):
+        """
+        Decode all input images into ndarray.
+
+        Note: This implementation doesn't properly handle error case in batch mode,
+        If one of the input image is corrupted, all request in the batch will fail.
```
Comment (aaronmarkham) [inline examples/model_service_template/mxnet_vision_service.py]:
```suggestion
        If one of the input images is corrupted, all requests in the batch will fail.
```

File: examples/model_service_template/mxnet_vision_service.py [hunk 2]
```diff
@@ -25,6 +26,15 @@ class MXNetVisionService(MXNetModelService):
     """
 
     def preprocess(self, request):
+        """
+        Decode all input images into ndarray.
+
+        Note: This implementation doesn't properly handle error case in batch mode,
```
Comment (aaronmarkham) [inline examples/model_service_template/mxnet_vision_service.py]:
```suggestion
        Note: This implementation doesn't properly handle error cases in batch mode,
```

File: examples/model_service_template/mxnet_vision_service.py (hunk 1 above)
Comment (frankfliu) [reply examples/model_service_template/mxnet_vision_service.py]: Done

File: examples/model_service_template/mxnet_vision_service.py (hunk 2 above)
Comment (frankfliu) [reply examples/model_service_template/mxnet_vision_service.py]: Done

**Pull request merged**